deployer/src/deployment/deploy_layer.rs

```diff
@@ -449,10 +454,24 @@ mod tests {
         let tmp_dir = Builder::new().prefix("shuttle_run_test").tempdir().unwrap();
         let path = tmp_dir.into_path();
 
+        let channel = Endpoint::try_from(logger_uri.to_string())
+            .unwrap()
+            .connect()
+            .await
+            .expect("failed to connect to logger");
+
+        let channel = ServiceBuilder::new()
+            .layer(ClaimLayer)
+            .layer(InjectPropagationLayer)
```

Comment:
Hmm, we should be fine without these layers in the tests.

Reply:
e2edeac07a50b5166c7f2fb1c8332a65020bc2d4

Reply:
The runtime manager expects a client with those layers, so we'll need to keep them (or refactor the runtime manager to create the client instead). I think we discussed removing the claim layer too, right @chesedo?

Reply:
Actually the server wants those layers. Since we will mock / stub the server, we can control the layers needed in each test